Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml

```diff
@@ -0,0 +1,344 @@
+- type: entity
+  parent: PlayerCyborgBase
+  id: PlayerCyborgEngineer
+  name: engineer cyborg
+  components:
+  - type: Sprite
+    drawdepth: Mobs
+    netsync: false
+    sprite: SimpleStation14\Mobs\Silicon\engineer_cyborg.rsi
+    layers:
+    - map: ["enum.DamageStateVisualLayers.Base"]
+      state: robot
+    - state: robot-flashlight
+      shader: unshaded
+      visible: false
+      map: [ "enum.ToggleVisuals.Layer" ]
```

Comment:
Dont know what the base looks like, but only the sprite path looks like it needs to be redefined, everything else would be on the parent